READMEs/install.md

```diff
@@ -60,4 +60,7 @@ Thanks to Brownie, ocean.py treats each Ocean smart contract as a Python class,
 
 You've now installed Ocean, great!
 
-Your next step is to [setup locally](setup-local.md).
+As a MacOS or Windows user, your next step is to [setup remotely](setup-remote.md).
+
+As a Linux user, your next step is to [setup locally](setup-local.md).
```

Comment:
The "default" flow is supposed to be to setup locally. Setup remote is supposed to come later.

We only need to make changes for a temporary workaround.

Accordingly, my suggestion for the changes is either:
- don't make _any_ changes, since "setup local" redirects the user at the very top anyway
- flip around the order of the changes above, and mention that for Mac/Win it's a workaround because of barge/local issues.

Reply:
I feel that the redirection at the top of the readme could easily be missed by skimming the content. Therefore, I recommend that we follow the second option.

Reply:
OK. Please go ahead with that:)